FootballStats/FootballStats/Repository/ScheduleRepository.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ScheduleRepository.swift
+//  FootballStats
+//
+//  Created by Hoang Linh Nguyen on 4/10/2023.
+//
+
+import Foundation
+import Alamofire
+import RxSwift
+
+protocol ScheduleRepository {
+    func requestSchedule() -> Observable<[FixtureResponse]>
+}
+
+final class ScheduleRepositoryImpl: ScheduleRepository {
+    func requestSchedule() -> Observable<[FixtureResponse]> {
+        let footballHeaders: HTTPHeaders = [
+            "X-RapidAPI-Key": Constants.APIKey,
+            "X-RapidAPI-Host": Constants.APIFootballHost
```

Comment:
@hlinh88 
```suggestion
        let footballHeaders: HTTPHeaders = [
            "X-RapidAPI-Key": Constants.APIKey,
            "X-RapidAPI-Host": Constants.APIFootballHost
```
Mình nên move những cái dùng chung nhiều lần này vào file

`FootballStats/FootballStats/Data/Source/API/APIService.swift
`